input/templates/partials/catalog/quickview.hbs

```diff
@@ -57,3 +58,4 @@
     </div>
   </div>
 </div>
+<!-- end catalog/quickview.hbs -->
```

Comment:
This is autogenerated too, so please remove it or we will end up with duplicated lines ;)